mlte/spec/spec.py

```diff
@@ -124,6 +112,11 @@ def _to_property_model(self, property: Property) -> PropertyModel:
         }
         return property_model
 
+    @classmethod
+    def load(cls, identifier: str = DEFAULT_SPEC_ID) -> Artifact:
```

Comment:
This should still fit in with the proposal above.

Reply:
Yes, for load(). Do note, however, that load_with has 3 params: id, context, and store. If we provide a default value for the id argument, what happens with context and store?

Reply:
Btw, one way to handle this would simply to force all of them to be named parameters, which would allow us to have a default value only for identifier.